GameServerLib/Packets/INetworkIdManager.cs

```diff
@@ -0,0 +1,7 @@
+﻿namespace GameServerCore
+{
+    public interface INetworkIdManager
```

Comment:
Why have an interface for our NetID manager? The interface will only ever be implemented by the actual manager, so there's no point in having it.
Please remove the interface. (and revert your other changes related to it)

Reply:
I think that scripts should never use NetID and other lib related stuff, so we don't need interface for that and should manage things behind the scenes.